scripts/debcheck

```diff
@@ -20,7 +20,7 @@ PUPPET_VERSION=${PUPPET_VERSION:-6}
 # user repos TBD
 
 case ${DEBIAN_RELEASE} in
-  focal)
+  jammy)
```

Comment:
This came as a bit of a surprise, but I guess it makes sense

Reply:
Yeah, I don't think we use that script anywhere right now (but we should!), and it was never updated to support both focal *and* jammy… now it got updated at least a bit :)